src/components/Seedphrase.jsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react'
 import './Seedphrase.css'
 
-export default function Seedphrase({ seedphrase, isBlurred = true }) {
+export default function Seedphrase({ seedphrase, isBlurred = true, centered = false }) {
   return (
-    <div className="seedphrase slashed-zeroes d-flex flex-wrap">
+    <div
+      className={`seedphrase slashed-zeroes d-flex flex-wrap ${
```

Comment:
We should consider using [classnames](https://www.npmjs.com/package/classnames) for this. It's very lightweight and makes conditionally joining classNames a bit easier.

Eg, this could be written as
```
<div
  className={classNames('seedphrase slashed-zeroes d-flex flex-wrap', {
    'justify-content-center align-items-center': centered,
  })}
>
```